test/common/testlib.py

```diff
@@ -634,8 +638,6 @@ def allow_restart_journal_messages(self):
 
     def allow_authorize_journal_messages(self):
         self.allow_journal_messages("cannot reauthorize identity.*:.*unix-user:admin.*",
-                                    "Error executing command as another user: Not authorized",
-                                    "This incident has been reported.",
```

Comment:
Are you completely sure that they can be dropped here? E. g. `check-connection TestConnection.testBasic()` logs in without calling `login_and_go()` but calls `allow_authorize_journal_messages()`.